### ISSUE TITLE
1.37.1: geany source tree is not autoconf 2.71 compliant

### ISSUE THREAD

**Issue by kloczek** [geany/geany#2771]
```
+ cd geany-1.37.1
+ autoreconf -fiv
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'build-aux'.
libtoolize: copying file 'build-aux/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
autoreconf: running: intltoolize --copy --force
ERROR: 'IT_PROG_INTLTOOL' must appear in configure.ac for intltool to work.
autoreconf: error: intltoolize failed with exit status: 1
```
BTW: Looks like geany is using a lot of custom macros which cold be replaced by standard one.

**Comment by kugel-** [comment]
Fixed via #2761. SCNR :-)

**Comment by kloczek** [comment]
OK closing :)
